Modules/Chatroom/classes/class.ilObjChatroomAccess.php

```diff
@@ -80,25 +78,19 @@ private static function checkRoomAccess(string $a_permission, int $a_ref_id, int
 
         switch ($a_permission) {
             case 'visible':
```

Comment:
Duplicated code. The cases `visible` and `read` are the same, can be simplified with:

```suggestion
...
            case 'visible':
            case 'read':
                if (!$is_online) {
                    $DIC->access()->addInfoItem(
                        ilAccessInfo::IL_NO_OBJECT_ACCESS,
                        $DIC->language()->txt('offline')
                    );
                    return false;
                }
                break;
....
```